libi2pd/FS.cpp

```diff
@@ -251,8 +256,22 @@ namespace fs {
 			auto p = root + i2p::fs::dirSep + prefix1 + chars[i];
 			if (boost::filesystem::exists(p))
 				continue;
-			if (boost::filesystem::create_directory(p))
+#ifdef TARGET_OS_SIMULATOR
```

Comment:
From what I found, in `TargetConditionals.h` `TARGET_OS_SIMULATOR` can be defined as `0`, which counts as _defined_ for `#ifdef`. I think `#if` should be used here instead. @vovasty please recheck.
https://github.com/xybp888/iOS-SDKs/blob/6d733d8ec34d59a9b13d8e374b199bd3a6630a39/iPhoneOS13.0.sdk/usr/include/TargetConditionals.h#L143

Reply:
Fixed. Thanks of spotting this!